src/nodes.coffee

```diff
@@ -2523,7 +2535,7 @@ exports.Arr = class Arr extends Base
       for fragment, fragmentIndex in answer
         if fragment.isHereComment
           fragment.code = "#{multident(fragment.code, o.indent, no)}\n#{o.indent}"
-        else if fragment.code is ', ' and not fragment?.isElision and fragment.type isnt 'StringLiteral'
+        else if fragment.code is ', ' and not fragment?.isElision and fragment.type not in ['StringLiteral', 'StringWithInterpolations']
```

Comment:
Luckily a test happened to catch this weird case where `Arr` is sniffing its fragments and deciding whether a fragment whose text is `', '` is an array comma that it wants to reformat or part of a string